packages/drivers/odsp-driver/src/contracts.ts

```diff
@@ -36,6 +36,10 @@ export interface IOdspResolvedUrl extends IFluidResolvedUrl {
     summarizer: boolean;
 
     sharingLink?: string;
+
+    containerPackage?: {
```

Comment:
maybe the shape here should be a bit more generic so we could potentially extend this some day, rather than container package?

Perhaps codeHint: { packageName: string } ?

With comments describing what this is for.